### PR TITLE
Allow modify whiskers mode for dataset

### PR DESCRIPTION
**Pull request by yngvarr992**

### CODE REVIEW

**Comment by abm0** [comment]
Hey @sgratzl!
Default 'nearest' whiskers mode config was not working for my data as expecting, making max whisker value lower than q3 value, so I've decided to allow it's configuration for datasets. What do you think? Please review.

File: src/data.ts [hunk 1]
```diff
@@ -135,6 +135,8 @@ export interface IBaseOptions {
    * @default 7
    */
   quantiles?: QuantileMethod;
+
+  whiskersMode?: 'nearest' | 'exact';
```
Comment (sgratzl) [inline src/data.ts:139]: can you add a comment similar to the previous ones what this option does?

**Review by sgratzl — CHANGES_REQUESTED** [review]

File: src/data.ts (hunk 1 above)
Comment (abm0) [reply src/data.ts:146]: Sure! Done.

**Comment by abm0** [comment]
Thanks!

**Pull request merged**